src/metric_definitions.go

```diff
@@ -2424,42 +2352,401 @@ var oracleSysMetrics = oracleMetricGroup{
 			metricType:    metric.GAUGE,
 			defaultMetric: true,
 		},
+		{
+			name:          "db.capturedUserCalls",
+			identifier:    "Captured user calls",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.executeWithoutParseRatio",
+			identifier:    "Execute Without Parse Ratio",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.logonsPerSecond",
+			identifier:    "Logons Per Sec",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.physicalReadBytesPerSecond",
+			identifier:    "Physical Read Bytes Per Sec",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.physicalReadIORequestsPerSecond",
+			identifier:    "Physical Read IO Requests Per Sec",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.physicalReadsPerSecond",
+			identifier:    "Physical Reads Per Sec",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.physicalWriteBytesPerSecond",
+			identifier:    "Physical Write Bytes Per Sec",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
+		{
+			name:          "db.physicalWritesPerSecond",
+			identifier:    "Physical Writes Per Sec",
+			metricType:    metric.GAUGE,
+			defaultMetric: false,
+		},
 	},
-	metricsGenerator: func(rows database.Rows, metrics []*oracleMetric, metricsChan chan<- newrelicMetricSender) error {
-		var sysScanner struct {
-			instID     int
-			metricName string
-			value      float64
-		}
+	metricsGenerator: rowMetricsGenerator,
+}
 
-		for rows.Next() {
+var oraclePDBSysMetrics = oracleMetricGroup{
+	name: "sys_metrics",
```

Comment:
```suggestion
	name: "pdb_sys_metrics",
```

`sys_metrics` group name is already used in [oracleSysMetrics](https://github.com/newrelic/nri-oracledb/blob/master/src/metric_definitions.go#L1618), could we use another name so the [skipGroup](https://github.com/newrelic/nri-oracledb/blob/master/src/metrics.go#L121) can be used if configured?

Reply:
Thinking of having minimal config setup as a customer ,so didnt go with the skipGroup implementation. Also it would mean the skipgroup needs to be set by every customer not using Container Database or need PDB.  Hence went with the flag implementation.

Reply:
Of course, I agree the flag implementation is the best choice. However, I would use a different group name to keep it consistent with the implementation (PDB metrics are in a different metric group) and to keep the skipGroup (which is already implemented) working.

Reply:
sure will change that